install/migrations/update_10.0.0_to_10.0.1/configs.php

```diff
@@ -41,6 +41,19 @@
 use Glpi\Agent\Communication\AbstractRequest;
 
 $migration->displayMessage('Add new configurations / user preferences');
+$migration->addPreQuery(
+    $DB->buildUpdate(
+        Config::getTable(),
+        [
+            'context' => 'inventory'
+        ],
+        [
+            'name' => 'inventory_frequency',
+            'context' => 'core'
+        ]
+    )
+);
```

Comment:
Is this really necessary ?

Reply:
Yes for users that have already run the migration... :/